scripts/heat/2-instances-pinging.yaml

```diff
@@ -0,0 +1,171 @@
+
+  west_net_pool_start:
+    type: string
+    description: Start of private network IP address allocation pool
+    default: 172.20.1.10
+
+  west_net_pool_end:
+    type: string
+    description: End of private network IP address allocation pool
+    default: 172.20.1.99
+
+resources:
+
+  private_net_east:
+    type: OS::Neutron::Net
+    properties:
+      name: private_net_east
+
+  private_subnet_east:
+    type: OS::Neutron::Subnet
+    properties:
+      network_id: { get_resource: private_net_east }
+      cidr: { get_param: east_net_cidr }
+      allocation_pools:
+        - start: { get_param: east_net_pool_start }
+          end: { get_param: east_net_pool_end }
+
+  private_net_west:
+    type: OS::Neutron::Net
+    properties:
+      name: private_net_west
+
+  private_subnet_west:
+    type: OS::Neutron::Subnet
+    properties:
+      network_id: { get_resource: private_net_west }
+      cidr: { get_param: west_net_cidr }
+      allocation_pools:
+        - start: { get_param: west_net_pool_start }
+          end: { get_param: west_net_pool_end }
+
+  router:
+    type: OS::Neutron::Router
+    properties:
+      external_gateway_info:
+        network: { get_param: public_net }
+
+  router_interface_east:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: router }
+      subnet_id: { get_resource: private_subnet_east }
+
+  router_interface_west:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: router }
+      subnet_id: { get_resource: private_subnet_west }
+
+  security_group:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      rules:
+        - direction: ingress
+          ethertype: IPv4
+
+  east_port:
+    type: OS::Neutron::Port
+    properties:
+      security_groups:
+        - { get_resource: security_group }
+      network_id: { get_resource: private_net_east }
+      fixed_ips:
+        - subnet_id: { get_resource: private_subnet_east }
+
+  east_floating_ip:
+    type: OS::Neutron::FloatingIP
+    properties:
+      floating_network: { get_param: public_net }
+      port_id: { get_resource: east_port }
+
+  west_port:
+    type: OS::Neutron::Port
+    properties:
+      security_groups:
+        - { get_resource: security_group }
+      network_id: { get_resource: private_net_west }
+      fixed_ips:
+        - subnet_id: { get_resource: private_subnet_west }
+
+  west_floating_ip:
+    type: OS::Neutron::FloatingIP
+    properties:
+      floating_network: { get_param: public_net }
+      port_id: { get_resource: west_port }
+
+  east:
+    type: OS::Nova::Server
+    properties:
+      name: east
+      image: { get_param: image }
+      flavor: { get_param: flavor }
+      key_name: { get_param: keypair }
+      networks:
+        - port: { get_resource: east_port }
+      user_data_format: RAW
+      user_data:
+        str_replace:
+          template: { get_file: /root/scripts/heat/ping_template.sh }
```

Comment:
Yes, but the point is this command is executed from qa_crowbarsetup (see 2nd commit), at that point I'm already using path for the template itself